WorkloadManagementSystem/PilotAgent/dirac-pilot.py

```diff
@@ -40,12 +40,42 @@
   pilotParams.pilotRootPath = os.getcwd()
   pilotParams.pilotScript = os.path.realpath( sys.argv[0] )
   pilotParams.pilotScriptName = os.path.basename( pilotParams.pilotScript )
+  grid = pilotParams.site.split( '.' )[0]
   log.debug( 'PARAMETER [%s]' % ', '.join( map( str, pilotParams.optList ) ) )
+  try:
+    import json
+  except ImportError:
+    log.error( 'No json module available, could not get pilot commands at runtime. Using the default list.' )
```

Comment:
Fine, but then you should not keep going from here: json is requested few lines below.